app/pubhub/[...slug]/route.ts

```diff
@@ -0,0 +1,38 @@
+import { notFound } from "next/navigation"
+import { NextRequest } from "next/server"
+
+import { createClientAsync } from "@/lib/soap/publizon/v2_7/generated/getlibraryuserorderlist"
+
+const resources = {
+  "/pubhub/v1/user/loans": {
+    GET: {
+      all: async () => {
+        const client = await createClientAsync(
+          "./lib/soap/publizon/v2_7/wsdl/getlibraryuserorderlist.wsdl"
+        )
+        return await client.GetLibraryUserOrderListAsync({
+          cardnumber: "1234567890",
```

Comment:
These will be dynamic eventually, depending on the token the API receives, right?